### ISSUE TITLE
Repo ownership

### ISSUE THREAD

**Issue by simonihmig** [kaliber5/ember-cli-yadda#84]
Hey @albertjan,

as we (@kaliber5) are still heavily invested into this addon for our projects, I would like to ask you what your thoughts are for moving the repo to our org? (as touched in https://github.com/albertjan/ember-cli-yadda/pull/81#issuecomment-561563819)

This would make it more justifiable to spent more time on the addon, for e.g.
* regular maintenance (dependency updates)
* some modernization (e.g. Github actions, dependabot etc.)
* architectural changes (e.g. new step definition format, as touched in https://github.com/albertjan/ember-cli-yadda/pull/81)

Of course this would not change the way how existing or new contributors would be able to participate if there is interest, everybody is certainly highly welcome!

/cc @lolmaus

**Comment by albertjan** [comment]
I tried moving it to @kaliber5 but that didn’t work because I wasn’t allowed. I can transfer ownership to you and then you can move it on.

**Comment by simonihmig** [comment]
We are currently discussing our approach, and maybe (probably?) this will lead to having a separate addon, as there will be too many breaking changes, and providing backwards compatibility would be too much trouble, especially als obviously the number of active users of this addon is limited. (anyone other than us using this actively? 😝)

So this addon could at least continue to live and have some minimal maintenance, for "legacy" projects, while the new one will take a more radical change...

Will keep you updated, but for now sorry for (prematurely) bothering you!

**Comment by albertjan** [comment]
I just tried moving it to you but I can’t because you have a fork it’s not allowed 😂. Let me know when you’ve removed it.

**Comment by simonihmig** [comment]
> I just tried moving it to you but I can’t because you have a fork it’s not allowed 😂. Let me know when you’ve removed it.

@albertjan I just did that!

As mentioned in the other comment, we (or rather @lolmaus) are currently working on a related but different addon for yadda integration. But as we still rely on this one for existing projects, and the recent PRs I did today were a PITA, as Travis is unbearably slow today, and I have no admin rights to switch to Github Actions, I would appreciate if we could do the ownership move asap so some maintenance and administration is easier and does not require me bugging you... 

Thanks again!
